### PR TITLE
Nipope/callout positioning

### LINKED ISSUE #28954: [Bug]: Dropdown callout renders to the right when list content exceeds vertical space
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz
    Memory: 16.38 GB / 31.59 GB
  Browsers:
    Edge: Spartan (44.22621.2134.0), Chromium (115.0.1901.203)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codepen.io/nipope1/pen/PoXqPbQ

## Actual Behavior
Callout renders to the right of dropdown when list content exceeds vertical space. When the width of the dropdown is equal to the width of the screen, the callout takes up the entire screen when opened (2nd example in attached Codepen).

## Expected Behavior
Callout should render below or (if necessary) above dropdown with a scrollbar if content exceeds vertical space

### Logs

_No response_

### Requested priority

Blocking

### Products/sites affected

Azure Deployment Environments (Azure Portal extension)

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by nipope**
## Previous Behavior

N/A, new feature

## New Behavior

Added new prop `edgeAxisFixed` to ICalloutProps. When `edgeAxisFixed` is true, the positioning logic will only try edges on the same axis as the directionalHint (i.e. top/bottom vs. left/right). 

Ex 1: If `directionalHint=DirectionalHint.leftTopEdge` and `edgeAxisFixed=true`, positioning logic will only attempt to reposition the callout to the right edge (if the left edge is not in bounds). It will not try to reposition the callout to the top or bottom edges.
Ex 2: If `directionalHint=DirectionalHint.bottomAutoEdge` and `edgeAxisFixed=true`, positioning logic will only attempt to reposition the callout to the top edge (if the bottom edge is not in bounds). It will not try to reposition the callout to the left or right edges.

## Related Issue(s)

- Fixes #28954

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3241bc93cbf65105b6843c2fc9e5b82e6c5d20bc'>3241bc93cbf65105b6843c2fc9e5b82e6c5d20bc</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react/src/components/Callout/Callout.types.ts [hunk 1]
```diff
@@ -214,6 +214,14 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement>, Rea
    */
```
Comment (fabricteam) [inline packages/react/src/components/Callout/Callout.types.ts:214]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29118&commitId=bb96c610278f39aaace0cd88e508434367cafc72&env=prod&clientType=FLUENTUI)


  <details><summary><b>ContextualMenu</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With submenu - RTL.hover.chromium.png|1000| Changed|
|ContextualMenu.With submenu - RTL.click.chromium.png|1000| Changed|

</details>


  <details><summary><b>Pivot - Overflow</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|4448| Changed|

</details>


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.WideIllustration.default.chromium.png|581| Changed|
|TeachingBubble.SmallHeadline.default.chromium.png|1074| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|382| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.default.chromium.png|206| Changed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|504| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic.hover.chromium.png|3332| Changed|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|4522| Changed|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|2864| Changed|
|react-charting-AreaChart.Multiple.hover.chromium.png|4208| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|3173| Changed|
|react-charting-LineChart.Basic.hover.chromium.png|2629| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/BaseButton-mount_base.html">643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/BaseButton-mount.html">626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Breadcrumb-mount_base.html">1707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Breadcrumb-mount.html">1698</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Checkbox-mount_base.html">1712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Checkbox-mount.html">1698</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/CheckboxBase-mount_base.html">1466</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/CheckboxBase-mount.html">1480</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ChoiceGroup-mount_base.html">2876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ChoiceGroup-mount.html">2871</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ComboBox-mount_base.html">646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ComboBox-mount.html">644</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/CommandBar-mount_base.html">6152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/CommandBar-mount.html">6152</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ContextualMenu-mount_base.html">12111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ContextualMenu-mount.html">20137</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DefaultButton-mount_base.html">740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DefaultButton-mount.html">732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRow-mount_base.html">2181</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRow-mount.html">2159</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRowFast-mount_base.html">2221</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRowFast-mount.html">2207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRowNoStyles-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DetailsRowNoStyles-mount.html">2008</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Dialog-mount_base.html">2606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Dialog-mount.html">2761</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DocumentCardTitle-mount_base.html">226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/DocumentCardTitle-mount.html">237</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Dropdown-mount_base.html">2035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Dropdown-mount.html">1999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/FocusTrapZone-mount_base.html">1131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/FocusTrapZone-mount.html">1124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/FocusZone-mount_base.html">1043</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/FocusZone-mount.html">1075</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-mount_base.html">41475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-mount.html">41738</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-virtual-rerender_base.html">19847</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-virtual-rerender.html">19767</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedList-virtual-rerender-with-unmount.html">50699</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-mount_base.html">230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-mount.html">228</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-virtual-rerender_base.html">212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-virtual-rerender.html">206</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">228</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/IconButton-mount_base.html">1113</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/IconButton-mount.html">1075</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Label-mount_base.html">324</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Label-mount.html">335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Layer-mount_base.html">2726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Layer-mount.html">2797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Link-mount_base.html">372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Link-mount.html">399</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/MenuButton-mount_base.html">936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/MenuButton-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/MessageBar-mount_base.html">21458</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/MessageBar-mount.html">21547</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Nav-mount_base.html">1925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Nav-mount.html">1952</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/OverflowSet-mount_base.html">773</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/OverflowSet-mount.html">774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Panel-mount_base.html">2045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Panel-mount.html">1740</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Persona-mount_base.html">740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Persona-mount.html">746</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Pivot-mount_base.html">856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Pivot-mount.html">858</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/PrimaryButton-mount_base.html">889</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/PrimaryButton-mount.html">870</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Rating-mount_base.html">4677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Rating-mount.html">4572</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SearchBox-mount_base.html">905</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SearchBox-mount.html">910</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Shimmer-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Shimmer-mount.html">1893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Slider-mount_base.html">1319</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Slider-mount.html">1307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SpinButton-mount_base.html">2833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SpinButton-mount.html">2909</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Spinner-mount_base.html">386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Spinner-mount.html">386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SplitButton-mount_base.html">1800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SplitButton-mount.html">1889</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Stack-mount_base.html">410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Stack-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/StackWithIntrinsicChildren-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/StackWithIntrinsicChildren-mount.html">857</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/StackWithTextChildren-mount_base.html">2605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/StackWithTextChildren-mount.html">2561</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SwatchColorPicker-mount_base.html">6159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/SwatchColorPicker-mount.html">6157</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/TagPicker-mount_base.html">1445</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/TagPicker-mount.html">1430</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Text-mount_base.html">373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Text-mount.html">363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/TextField-mount_base.html">926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/TextField-mount.html">936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-mount_base.html">827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-mount.html">816</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-virtual-rerender_base.html">602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-virtual-rerender.html">578</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Toggle-mount_base.html">620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/Toggle-mount.html">604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/buttonNative-mount_base.html">194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29118/perf-test/buttonNative-mount.html">187</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by layershifter** [comment]
Closing for housekeeping 🏡  Feel free to re-open 👍

**Pull request closed without merging**